libkbfs/md_journal.go

```diff
@@ -464,6 +464,17 @@ func (j *mdJournal) convertToBranch(
 	j.j = tempJournal
 	j.branchID = bid
 
+	// Garbage-collect the old non-branch entries.  TODO: we'll
```

Comment:
Actually, I think you don't need this after all. See line 462, and the referred-to defer (which nukes `journalTempDir`.)